dotCMS/src/main/java/org/apache/velocity/tools/view/servlet/ServletToolboxManager.java

```diff
@@ -150,45 +151,37 @@ public void registerService () {
      * This method will ensure there is exactly one ServletToolboxManager
      * per xml toolbox configuration file.
      */
-    public static synchronized ServletToolboxManager getInstance(ServletContext servletContext,
-                                                                 String toolboxFile)
+    public static synchronized ServletToolboxManager getInstance(final ServletContext servletContext,

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use block level rather than method level synchronization](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)